frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelSecondaryTabs.tsx

```diff
@@ -24,6 +25,18 @@ export function FunnelSecondaryTabs(): JSX.Element | null {
                 <h4 className="secondary">Options</h4>
                 <FunnelConversionWindowFilter />
             </Card>
+            <Card style={{ marginTop: 16 }} data-attr="funnel-exclusion-options">
+                <h4 className="secondary">
+                    Exclusions
+                    <Tooltip
+                        placement="right"
+                        title="Exclude users who did not complete specific events from your funnel, or specific steps of your funnel."
```

Comment:
Yeah correct, we exclude users who did specific events (the exclusion events) between the funnel steps.

To answer your earlier question: 
> Noticed some weird behavior (though not sure if this is Core Analytics scope, CC @neilkakkar). Tested using vanilla demo data. Without the exclusion step my universe is 43 users, yet when I apply the exclusions step, not only are users excluded from converting to step 2, but also they're removed from consideration since step 1.

The users are indeed removed from consideration, that's intended, as otherwise the conversion rates get messed up (counted in step 1, but forced to eject from step 2 because of exclusion)